debezium-server-core/pom.xml

```diff
@@ -56,6 +56,16 @@
             <artifactId>quarkus-junit5</artifactId>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>io.quarkus</groupId>
+            <artifactId>quarkus-junit5-internal</artifactId>
+            <scope>test</scope>
+        </dependency>
```

Comment:
This dependency is going to be problematic it seems. It provides means to nicely test log output, however it also causes the DebeziumServerTest#testProps to fail with

```
org.opentest4j.AssertionFailedError: 
Expecting:
 <"inventory.customers">
to be equal to:
 <"public.table_name">
but was not.
Expected :public.table_name
Actual   :inventory.customers
```

Reply:
Ok, false alarm the cause might be in failsafe configuration